Description: Remove the Update Assets menu entry from the Online Updater
 We build the retroarch-assets package to provide these files and updates
 should be handled by APT as they are installed under /usr. Eliminating this
 entry removes the potential for confusion about this issue with users.
Author: Peter J. Mello <admin@petermello.net>
Origin: other
Forwarded: no
Last-Update: 2025-07-02

--- a/menu/menu_displaylist.c
+++ b/menu/menu_displaylist.c
@@ -15111,14 +15111,6 @@ bool menu_displaylist_ctl(enum menu_disp
                }
 #endif
 
-#ifdef HAVE_UPDATE_ASSETS
-               if (menu_entries_append(info->list,
-                        msg_hash_to_str(MENU_ENUM_LABEL_VALUE_UPDATE_ASSETS),
-                        msg_hash_to_str(MENU_ENUM_LABEL_UPDATE_ASSETS),
-                        MENU_ENUM_LABEL_UPDATE_ASSETS,
-                        MENU_SETTING_ACTION, 0, 0, NULL))
-                  count++;
-#endif
 #if !defined(_3DS)
                if (menu_entries_append(info->list,
                         msg_hash_to_str(MENU_ENUM_LABEL_VALUE_UPDATE_AUTOCONFIG_PROFILES),
